jetstream/metrics/__init__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Google LLC
```

Comment:
Could you move this to `jetstream/core/metrics.py` and put the metrics into a class with methods?

Reply:
You could init the object in `server_lib.py` or in the driver init, and call it's methods to collect different metrics? 1 server only has 1 driver with 1 jetstream_metrics obj?

Reply:
+1 for moving code to metrcis.py insteand of __init__.py.  You can use Singleton pattern if only need init once (https://en.wikipedia.org/wiki/Singleton_pattern)